cmd/planet-exporter/internal/internal.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"planet-exporter/collector"
 	taskdarkstat "planet-exporter/collector/task/darkstat"
+	"planet-exporter/collector/task/ebpf"
```

Comment:
btw could you name this import as `taskebpf "planet-exporter/collector/task/ebpf"`?

Reply:
I was trying to follow golang style guidelines :)
[CodeReviewComments#imports](https://github.com/golang/go/wiki/CodeReviewComments#imports)
Renamed for now for consistency, *maybe we can remove the renaming and follow best practices in future for all tasks. :)
Thanks!